spec/serializers/requests/index_serializer_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+RSpec.describe Request::IndexSerializer, type: :serializer do
+  let!(:requests) { 5.times { create(:request) } }
+  let(:serialization) do
+    ActiveModelSerializers::SerializableResource.new(
```

Comment:
this syntax is not familiar for me, could you explain what it does?

Reply:
It's basically a version of `each_serializer` that you use when sending off your collection as soon in a controller action.